packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -66,7 +67,9 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
 
   static GROUP_FILTER = ContextActions.groups.high + 50;
 
-  static GROUP_GOTO = ContextActions.groups.high + 51;
+  static GROUP_EXPAND_COLLAPSE = ContextActions.groups.high + 51;
+
+  static GROUP_GOTO = ContextActions.groups.high + 52;
```

Comment:
Could probably space these priorities out a bit so we don't need to adjust if we insert another. 


```suggestion
  static GROUP_EXPAND_COLLAPSE = ContextActions.groups.high + 55;

  static GROUP_GOTO = ContextActions.groups.high + 60;
```